config.json

```diff
@@ -1020,6 +1020,17 @@
         "algorithms",
         "math"
       ]
+    },
+    {
+      "slug": "dnd-character",
+      "uuid": "bd15a0e0-c19e-4a52-be6c-1a61d14c54cd",
+      "core": false,
+      "unlocked_by": "robot-simulator",
+      "difficulty": 3,
+      "topics": [
+        "random",
+        "structs"
```

Comment:
maybe also typespec?

Reply:
I definitely think it's worth discussing having students do typespecs, but I also think we should look at evaluating that for the track as a whole.

Reply:
Then let's leave as is for now, and eval as a whole later

Reply:
Then let's leave as is for now, and eval as a whole later